File: app/src/main/java/remix/myplayer/bean/mp3/Artist.java
package remix.myplayer.bean.mp3;

/**
 * Created by Remix on 2017/10/22.
 */

public class Artist {

  private int ArtistID;
  private String Artist;
  private int Count;

  public Artist(int artistID, String artist, int count) {
    ArtistID = artistID;
    Artist = artist;
    Count = count;
  }

  public int getCount() {
    return Count;
  }

  public void setCount(int count) {
    this.Count = count;
  }

  public int getArtistID() {
    return ArtistID;
  }

  public void setArtistID(int artistID) {
    ArtistID = artistID;
  }

  public String getArtist() {
    return Artist;
  }

  public void setArtist(String artist) {
    Artist = artist;
  }
}

File: app/src/main/java/remix/myplayer/bean/mp3/Artist.kt
package remix.myplayer.bean.mp3

import java.io.Serial

/**
 * Created by Remix on 2017/10/22.
 */

data class Artist(val artistID: Long,
                  val artist: String,
                  var count: Int) : APlayerModel {

  override fun getKey(): String {
    return artistID.toString()
  }

  companion object {
    @Serial
    private const val serialVersionUID: Long = -8314996381751820478L
  }
}
